### DISCUSSION TITLE
QField & QGis version

### DISCUSSION THREAD

**Discussion by bettellam** [opengisch/QField#3291] [Q&A]
Good morning,
I am looking for information regarding version alignment between QGis and QField.
In particular I am using both the QField master-dev version (c95671) and I wanted to know which version of QGis it is hooked to.
Recently my Pull Request has been merged for NMEA messages https://github.com/qgis/QGIS/pull/49820
... I wanted to understand when this update will also be available in QField
Thank you

**Comment by m-kuhn** [comment]
Even though this is not a fixed rule, QField often uses the latest QGIS release as base.

You can check the current version here: https://github.com/opengisch/QField/blob/master/vcpkg/overlay/qgis/portfile.cmake#L1

Most likely this patch will be available in QField whenever a new QField release is made after 3.28 has been released.

**Reply by bettellam** [reply]
Ok thank you
